src/main/java/edu/kis/powp/jobs2d/command/DriverCommandVisitor.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.Iterator;
+
+
+/**
+ * Visitor class for driver commands.
+ */
+public class DriverCommandVisitor implements Visitor {
+    private int counter = 0;
+
+    public int getCounter(){
+        return this.counter;
+    }
+
+    @Override
+    public void visit(DriverCommand driverCommand) {
+        counter++;
+        System.out.println("Counter: " + counter);

```

Comment:
Please remove such side effects. Printing the result obtained from getCounter will be enough.